_data/people.yml

```diff
@@ -71,7 +71,7 @@ allen_g:
   topics:
   email: gallen@ncsa.illinois.edu
   homepage: http://www.astro.illinois.edu/people/gdallen
-  
+
```

Comment:
I don't get it.. what is the problem here?

On 02.03.16 14:55, Torbjörn Klatt wrote:
> In _data/people.yml
> <https://github.com/JLESC/jlesc.github.io/pull/146#discussion_r54724803>:
> 
>> @@ -71,7 +71,7 @@ allen_g:
>>    topics:
>>    email: gallen@ncsa.illinois.edu
>>    homepage: http://www.astro.illinois.edu/people/gdallen
>> -  
>> +
> 
> You should configure your editor to keep extra spaces at end of line for
> the whole of the JLESC site. It will break Markdown!
> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/JLESC/jlesc.github.io/pull/146/files#r54724803>.
>